src/components/Navbar.jsx

```diff
@@ -1,44 +1,44 @@
-import { useState } from 'react';
-import * as FaIcons from 'react-icons/fa';
-import * as AiIcons from 'react-icons/ai';
-import { Link } from 'react-router-dom';
-import { SidebarData } from './SidebarData';
-import './Navbar.css';
-import { IconContext } from 'react-icons';
+import UserProfile from "@/components/document/Header/UserProfile";
 
-function Navbar() {
-  const [sidebar, setSidebar] = useState(false);
+const NavbarSubSection = ({ subsectionName }) => {
+  return <div 
+    className='px-3 p-2 hover:bg-[#5C00B8] rounded'
+    onClick={(e) => {
+      print(e)
+      // TODO: Do some routing here
+    }}>
+    {subsectionName}
+  </div>
+}
 
-  const showSidebar = () => setSidebar(!sidebar);
+const NavbarSection = ({ title, subsections }) => {
+  return <div className=' text-lg font-bold py-3'>
+    {title}
+    <div className='ml-3 my-2 text-base font-light'>
+      <NavbarSubSection subsectionName={subsections[0]}/>
+      <NavbarSubSection subsectionName={subsections[1]}/>
+    </div>
+  </div>
 
+}
+
+function Navbar() {
   return (
     <>
-      <IconContext.Provider value={{ color: '#fff' }}>
-        <div className='navbar'>
-          <Link to='#' className='menu-bars'>
-            <FaIcons.FaBars onClick={showSidebar} />
-          </Link>
+      <div className='fixed w-[350px] h-screen bg-[#340068] text-white'>
```

Comment:
Same thing with the arbitrary values here - we probably want to change the bg colour to a tailwind theme, and then it would be optimal to use like `w-80` or something I think? It's 320px but it's specified in em which is better bc it's relative to the user's font size config